src/reference/command_output_tags.adoc

```diff
@@ -15,13 +59,15 @@ unexpected error conditions.
 [[COG_TEMPLATE]]`COG_TEMPLATE`::
 Renders the response with the template provided. For example, if your bundle
 includes a template named "instance-list" you would output `COG_TEMPLATE:
-instance-list` before outputting your response.  You'll also need to use the
-JSON tag for the template to be applied.
+instance-list` before outputting your response. If not used with JSON output
+expect the raw text output from your command to be available in the `$body`
+attribute of the first item of `$results` in the template.
 
 [[JSON]]`JSON`::
 Parses the following response as JSON. Must be printed on a new line, by
 itself, directly before your command's output. If not provided all non-tag
-output is assumed to be raw text.
+output is assumed to be raw text. Anything printed after this tag will be
+parsed as JSON, so you may not include any other tags after this one.
```

Comment:
Let's say "must not" instead of "may not", per [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt)